htdocs/class/xoopslocal.php

```
@@ -58,20 +58,12 @@ public static function substr($str, $start, $length, $trimmarker = '...')
-                    return $text;
-                }
-
-            }
+        $converted_text = mb_convert_encoding($text, 'UTF-8', 'auto');
+        if ($converted_text !== false && !is_array($converted_text)) {
+            return $converted_text;
         }
 
-        return utf8_encode($text);
+        return $text;
     }
```

PHPMD (2.15.0)

[error] The method utf8_encode is not named in camelCase. (undefined)

(CamelCaseMethodName)